src/commands/watch.rs

```diff
@@ -132,8 +132,14 @@ impl WatchCommand {
         let source_code_tx = Arc::new(source_code_tx);
         let manifest_tx = Arc::new(manifest_tx);
 
+        // Check if the --direct-mounts flag is set We can skip watching assets
+        // if we are direct mounting instead of a tempfs
```

Comment:
```suggestion
        // Check if the --direct-mounts flag is set. We can skip watching assets
        // if we are direct mounting instead of a tempfs
```

Nit: Seems like you used a plugin to wrap the comments but I think you want a period in that sentence.

Reply:
good catch. I simplified the comment as suggested by @itowlson